client/src/components/TOTP/Login.js

```diff
@@ -159,9 +159,7 @@ class Login extends Component {
             onChange={this.handleChangeCode}
             onKeyUp={this.handleInputKeyUp}
           />
-          {error && <div className="help-block">
-            {i18n._t('TOTPLogin.ERROR', 'Invalid code')}
```

Comment:
@ScopeyNZ this part is important - I'll reopen and rebase onto your PR once its merged

Reply:
Oh yep - I missed that bit.